### PR TITLE
Turn off update node labels

### PR DESCRIPTION
**Pull request by charliedmcb**
We no longer need this as it will be supported within aks instead. Dropping this to remove the kubectl calls on startup.

### CODE REVIEW

**Review by matthchr — APPROVED** [review]

**Comment by xuto2** [comment]
are we doing the labelling explicitly on RP side? How do we ensure the node-image-version label version is updated in a node image upgrade event?

File: parts/linux/cloud-init/artifacts/cse_main.sh [hunk 1]
```diff
@@ -176,7 +176,6 @@ configureSwapFile
 ensureSysctl
 ensureKubelet
 ensureJournal
-ensureUpdateNodeLabels
```
Comment (xuto2) [inline parts/linux/cloud-init/artifacts/cse_main.sh:179]: what about those scripts?

**Review by xuto2 — CHANGES_REQUESTED** [review]
Let's chat before merging this. I'm curious about the design change

**Comment by charliedmcb** [comment]
> Let's chat before merging this. I'm curious about the design change

@xuto2 I'd be happy to discuss the change in design. This change is partially a result of an ICM. We are planning to merge this in before the cutoff this Wednesday, so we can roll this out as soon as possible.

File: parts/linux/cloud-init/artifacts/cse_main.sh (hunk 1 above)
Comment (charliedmcb) [reply parts/linux/cloud-init/artifacts/cse_main.sh:174]: The reason I didn't do a revert of the previous change was due to the fact that I took care of the other label service, and didn't want it popping back up again

Comment (paulgmiller) [reply parts/linux/cloud-init/artifacts/cse_main.sh:174]: [10:17 AM] Charlie McBride
    The reason I didn't do a revert of the previous change was due to the fact that I took care of the other label service, and didn't want it popping back up again
(1 liked)

Comment (charliedmcb) [reply parts/linux/cloud-init/artifacts/cse_main.sh:174]: Added an issue to track code clean up for the service:
https://github.com/Azure/AgentBaker/issues/684

Comment (xuto2) [reply parts/linux/cloud-init/artifacts/cse_main.sh:174]: We can simply remove those service files to reduce garbage code

**Pull request merged**